README.md

```diff
@@ -1,5 +1,5 @@
 ![crates.io version](https://img.shields.io/crates/v/fluence?color=green)
 
-## Rust backend SDK
+## Marine Rust SDK
 
-This SDK intended to run backend application on the Fluence network. More information about usage and some internals could found in [docs](https://fluence-labs.readme.io/docs/rust-backend-sdk).
+This SDK is intended to run backend application on the Fluence network with [Marine](https://github.com/fluencelabs/marine) runtime. More information about usage and some internals could found in [docs](https://fluence-labs.readme.io/docs/rust-backend-sdk).
```

Comment:
Uh oh. Link to docs currently leads to the `fluence.dev` :( I'll try to fix that ASAP so it points to the old documentation, BUT. 

But maybe let's point to to https://doc.fluence.dev/docs/development_development for example?

Reply:
Seems that `https://fluence-labs.readme.io/docs/rust-backend-sdk` is now pointing to the old docs, so link is kinda fine. Ideally though we need to copy paste that doc to the latest documentation